test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -5108,6 +5109,9 @@ def revert_to_defaults(self, reverthooks=True, revertqueues=True,
                 reverthooks = False
             hooks = self.status(HOOK, level=logging.DEBUG)
             hooks = [h['id'] for h in hooks]
+            for h in skip_site_hooks:
+                hooks.remove(h)
+            hooks.remove('pbs_cgroups')
```

Comment:
I think pbs_cgroups got removed in the for loop, this is why travis is failing.
However, isn't this PR about not deleting the cgroup hook?

Reply:
Hi Prakash, I am deleting pbs_cgroups from hooks list which will be deleted in setUp.
Travis was failing due to extra line of code which I have removed now.

Reply:
@shilpakodli before calling `hooks.remove('pbs_cgroups')` you need to check whether hooks has 'pbs_cgroups' in it or not else it will throw `ValueError: list.remove(x): x not in list` error

Reply:
I missed seeing that you are removing it from the list being sent for deletion.

Reply:
@hirenvadalia - I also thought of that but assumed that pbs_cgroups will always be there. But, it is better to be safe than sorry.